arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -247,7 +262,21 @@
   "parameters": {
     "SubscriptionIdParameter": {
       "name": "subscriptionId",
-      "description": "Azure subscription id.",
+      "description": "The Azure subscription ID.",
+      "in": "path",
+      "type": "string",
+      "required": true
+    },
+    "ResourceGroupNameParameter": {
+      "name": "resourceGroupName",
+      "description": "Name of the resource group in which the web service is located.",
+      "in": "path",
+      "type": "string",
+      "required": true
```

Comment:
Please apply the extension `"x-ms-parameter-location": "method"`, for "ResourceGroupNameParameter" and "WebServiceNameParameter". By default parameters defined in the global parameters section will be properties of the client in the generated code. You would not want resourceGroupName and webServiceName as properties of the client as the customer would like to use the same client for making requests against different resource groups or webservices. Changing the property of the client everytime is inconvenient. Hence please apply the above mentioned extension to these two parameters.